libs/ui/elements/src/lib/pagination/pagination.component.html

```diff
@@ -0,0 +1,36 @@
+<div style="width: 100%; height: 4rem; display: flex; justify-content: center">
```

Comment:
no hard coded style when useless, use CSS class for this
`w-full, h-something, d-flex` etc...